inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -30,26 +30,39 @@
 
 
 @experiments.over('experiment_name', '^119..')
+@utils.for_each_value
 def experiment_name(self, key, value):
-    """Name of experiment."""
-    value = utils.force_list(value)
-    self.setdefault('affiliation', [v.get("u") for v in value if v.get('u')])
+    """Short experiment name."""
+    if 'u' in value:
+        self.setdefault('affiliations', [])
+        if isinstance(value['u'], str):
```

Comment:
Please check [Jan's comment](https://github.com/inspirehep/inspire-next/pull/1069) on using `six.text_type` instead of `str` for Python 2/3 compatibility.